### PR TITLE
Static e2e pages

### PR DESCRIPTION
**Pull request by svettwer**
Added a static sandbox page containing most HTML elements. This can be used for demos, user tutorials, quickly exploring some functionality etc.

### CODE REVIEW

**Comment by svettwer** [comment]
https://sakuli-github-io-git-static-e2e-pages.sakuli.now.sh/e2e-pages/sandbox

File: static/e2e-pages/sandbox/index.html [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+            </li>
+            <li>
+                <a href="http://instagram.com" title=" on Instagram">Instagram</a>
+            </li>
+            <li>
+                <a href="http://dribbbble.com" title=" on Dribbble">Dribbble</a>
+            </li>
+            <li>
+                <a href="http://github.com" title=" on Github">Github</a>
+            </li>
```
Comment (cliffle) [inline static/e2e-pages/sandbox/index.html:341]: I think those should be open in a new tab so we can use it as an example for switching tabs (sakuli/docs#31).

Comment (svettwer) [reply static/e2e-pages/sandbox/index.html:341]: Good point :+1:

**Comment by svettwer** [comment]
Yes, an IFrame would be good for demonstration but I agree that due to the size of a page, Sakuli would find anything instantaneously. I'll alter the [cookbook IFrame entry](https://sakuli.io/docs/cookbook/frames/) to mention that large websites might cause lacks in performance when IFrame Content is searched without a manual switch. That should make it clear that an automatic search is not always recommended.

File: static/e2e-pages/sandbox/index.html [hunk 2]
```diff
@@ -301,10 +291,8 @@ <h1>Title</h1>
 <section>
     <h1>Code examples</h1>
     <code>
-        <pre>
       sudo ipfw pipe 1 config bw 256KByte/s
       sudo ipfw add 1 pipe 1 src-port 3000
-    </pre>
```
Comment (cliffle) [inline static/e2e-pages/sandbox/index.html]: Without the `<pre>` tag, the code text does not seem to be formatted right.

File: static/e2e-pages/sandbox/index.html [hunk 3]
```diff
@@ -400,7 +386,6 @@ <h1>Random Stuff</h1>
           -rw-r--r--   1    2203 Jun  4 15:59 touch-icon-ipad-retina-precomposed.png
           -rw-r--r--   1    1046 Jun  4 15:59 touch-icon-iphone-precomposed.png
           -rw-r--r--   1    1779 Jun  4 15:59 touch-icon-iphone-retina-precomposed.png
-        </pre>
```
Comment (cliffle) [inline static/e2e-pages/sandbox/index.html]: Without the `<pre>` tag, the sample text does not seem to be formatted right

**Comment by svettwer** [comment]
Hmhm locally it looks okay... strange... :thinking:

**Comment by svettwer** [comment]
Ah, okay... Seems that my browser cached the state before the change somehow, my bad.

**Pull request merged**